turtle.py

```diff
@@ -3,6 +3,13 @@
 import time
 
 
+def turn_right():
+    global score
```

Comment:
According to main function, global score isn't needed.
So, I will coordinate your code and adopt.

Reply:
I think a score variable  should be global.
I modify main function.
Thank you.